frontends/web/src/routes/device/manage-backups/manage-backups.jsx

```diff
@@ -25,7 +26,23 @@ import * as styles from './manage-backups.css';
 
 @translate()
 export default class ManageBackups extends Component {
+    getDevice = () => {
+        if (this.props.deviceID === null) {
+            return null;
```

Comment:
As far as I can tell, these lines do not seem to be necessary. `object[null]` returns `undefined` for me and thus the return statement below already achieves the same.

Reply:
I prefer `null` over `undefined`, it is an explicit value for 'does not exist'.

I side-stepped the issue though by returning a boolean (don't actually need the device anywhere).